package.json

```diff
@@ -31,7 +31,7 @@
     "@types/fs-extra": "^9.0.13",
     "@types/jest": "^27.5.0",
     "@types/minimatch": "^3.0.5",
-    "@types/node": "^14",
+    "@types/node": "^12",
```

Comment:
hmm what's going on here? Node 12 has recently reached end of life. 

https://endoflife.date/nodejs

Reply:
Hmm, it was upgraded recently, downgraded during the merge from live, and restored on my commit from a while ago, running yarn default. Should be fine now.